ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-community-lists.j2

```diff
@@ -0,0 +1,15 @@
+{# eos - ip-community-lists #}
+{% if ip_community_lists is arista.avd.defined %}
+!
+{%     for community_list in ip_community_lists %}
+{%         for entry in community_list.entries %}
```

Comment:
Need to add a guard for ```entry.action```. My suggestion to change line5: ```for entry in community_list.entries if entry.action is arista.avd.defined```

Reply:
Similarly, please add guard for ```community_list.name``` in line4 or any way you like.

Reply:
action is required parameter, in future it can be controlled by schema, but the playbook must fail IMO if it's not present

Reply:
name is mandatory as well

Reply:
Even though it's mandatory. We do the arista.avd.defined tests currently.